packages/budget-it/src/components/AddCategory.vue

```diff
@@ -1,18 +1,23 @@
 <script setup lang="ts">
-  import { ref } from 'vue';
+  import { ref, computed } from 'vue';
+  import { useCategoriesStore } from '@/stores/categories';
 
-  const emit = defineEmits(['addCategory']);
   const categoryName = ref('');
   const isActive = ref(false);
+  const isDisabled = computed(() => (
+    !categoryName.value
+    || useCategoriesStore().categoryList.some(({ name }) => (
```

Comment:
Add this use case so the button is disabled when there is a duplicate and also to prevent the form from submitting when hitting the enter key.